app/mailers/cards_mailer.rb

```diff
@@ -0,0 +1,8 @@
+class CardsMailer < ActionMailer::Base
+  default from: 'info@flashcards.com'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.